Dms installer v1

b-f-chan opened the pull request:
To review & merge conflicts with @samrichca

b-f-chan commented:
This is a lot of content and will need a another round of review in detail.

For now, please sanity check for any major/glaring issues before deploy tomorrow.

**Netlify Link**:
https://60633cf66ced6f0de57f41f5--overture.netlify.app/

@anncatton - Just need you to review **Using the Data Portal**:
https://60633cf66ced6f0de57f41f5--overture.netlify.app/documentation/dms/user-guide/data-portal/

@christinayung - If you can have a look at the Introduction, and also sanity check the Installation & Configuration:
https://60633cf66ced6f0de57f41f5--overture.netlify.app/documentation/dms/
https://60633cf66ced6f0de57f41f5--overture.netlify.app/documentation/dms/installation/installation/

@blabadi - Sanity check of Installation Guide:
https://60633cf66ced6f0de57f41f5--overture.netlify.app/documentation/dms/installation/installation/

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+- **(A)** The list of available values for that field are displayed.  Clicking the checkmark beside a value will filter the data table to show only records that match that field value.
+
+- **(B)** For each of the fields available values, a count (or aggregation) is displayed to the right of it, indicating how many records in the daa set contain this value.
```
typo: `daa` -> `data`

b-f-chan replied:
fixed

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+<Warning>**NOTE:** If you deny access to the DMS platform, then your only alternative is to log in again with a different identity provider with which you are comfortable granting access.</Warning>
+
+4. After granting access, if the email address on your identity provider profile is not configured to be publicly-accessible, you may receive an error message indicating so:
```
can just be `publicly accessible`, no hyphen needed

b-f-chan replied:
fixed

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+- **(B)** For each of the fields available values, a count (or aggregation) is displayed to the right of it, indicating how many records in the daa set contain this value.
+
+- **(C)** Click the magnifying glass control to display a text search box that lets you search for a specific value for that fields.  This is useful when the list of values for a field is very long.
```
typo: `that fields` -> `that field`

b-f-chan replied:
fixed

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+- **(C)** Click the magnifying glass control to display a text search box that lets you search for a specific value for that fields.  This is useful when the list of values for a field is very long.
+
+3. As you select different filters, your query is dynamically built and the query parameters are displayed above the results table.  The results table itself, in the middle of the page, dynamic refreshes with updated results based on your query:
```
typo: `dynamic refreshes` -> `dynamically refreshes`

b-f-chan replied:
fixed

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+![Entity](../assets/filtered-data.png 'Filtered Data')
+
+4. You can use the **Columns** control in the top-right of the results table to hide or show field columns which you feel are not useful or useful to your exploration needs:
```
is the `or` needed here? maybe just `which you feel are not useful to your exploration needs`

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -0,0 +1,100 @@
+
+Once you have [logged in](#logging-into-the-portal), you can manage various aspects of your DMS user profile.
+
+To access your profile, click your display nane, then select **Profile & Token** in the top-right in the header.  
```
typo: `nane` -> `name`

anncatton reviewed (changes requested):
Just a couple minor things: 
1) do we want to mention logout?
2) maybe a mention + screenshot for an expired api token?
3) mention that generating a new token automatically revokes all the user's previous tokens
Also just some typos

b-f-chan commented:
@anncatton per your feedback also did the following:

- Added how to logout
- Added description and screenshot of expired token
- Added note that generating new token revokes all previous tokens

anncatton commented on markdown/documentation/dms/user-guide/data-portal.md:
```diff
@@ -97,4 +105,8 @@ A user can revoke their own API token from the Data Portal UI once they have log
+
+# Logging Out of the Portal
+
+To log out at any time, simply click **Logout** in the top right of the header.  Once logged out, you are automatically redirected to the Data Explorer by default.
```
probably don't need `by default` at the end here since you're already saying `automatically` earlier.

anncatton reviewed (approved):
Just a couple more small comments, otherwise lgtm on this section 👍

b-f-chan commented:
Implemented all of @blabadi's review comments including:

- Fix broken system requirements link in Installation section
- Correct note to indicate users can source the bash command to .bashrc themselves
- Add warning that no automatic backup of data volumes supported right now
- Add note for users to make sure Certbot setup for automatic renewal
- Add note that not all storage services need /data folder created
- Add instructions on how to check Minio browser
- Add warning current system is a single node system and not meant to be highly available

One exception - Have not included steps for making Arranger & ES private until we have determined and validated the steps.  Will add that as a priority item in next doc update/release.

New Netlify for verification:
https://60638bc5d33573290140c9a7--overture.netlify.app/documentation/dms/installation/installation/
